src/host/misc.cpp

```diff
@@ -115,7 +115,7 @@ BOOL CheckBisectStringW(_In_reads_bytes_(cBytes) const WCHAR* pwchBuffer,
 // - pwchBuffer - Pointer to Unicode string buffer.
 // - cWords - Number of Unicode string.
 // - cBytes - Number of bisect position by byte counts.
-// - fEcho - TRUE if called by Read (echoing characters)
+// - fPrintableControlChars - TRUE if called by Read (echoing characters)

```

Comment:
here three.

Reply:
i mean yes at this point i get it :wink:

Reply:
shhh my GH didn't update with your change. blame GH. also sometimes its hard to see multiple instances. safe > sorry.